tests/pytorch/test_cuda_graphs.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) 2022-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
+
+
+import argparse
+
+import torch
+import transformer_engine.pytorch as te
+import apex
```

Comment:
We should make this import optional (and then skip the tests that would rely on it).

Reply:
These aren't the finalized tests. Apex won't be used at all.